fastify.js

```diff
@@ -408,9 +410,15 @@ function build (options) {
     }
 
     validateBodyLimitOption(opts.jsonBodyLimit)
-    var jsonBodyLimit = opts.jsonBodyLimit || _fastify._jsonBodyLimit
 
-    _fastify.after((notHandledErr, done) => {
+    _fastify.after(afterRouteAdded)
```

Comment:
This should be kept the same. Defining the function separately even though this is the only place it is used makes the code less clear.

Reply:
I agree